packages/yoastseo/src/scoring/assessments/inclusiveLanguage/configuration/disabilityAssessments.js

```diff
@@ -446,8 +446,8 @@ const disabilityAssessments =  [
 	{
 		identifier: "to drive crazy",
 		nonInclusivePhrases: [ "crazy" ],
-		inclusiveAlternatives: "<i>to drive to one's limit, to get on one's last nerve, to make one livid, to aggravate, to make blood boil, " +
-			"to exasperate, to irritate to the limit</i>",
+		inclusiveAlternatives: "<i>to drive one to their limit, to get on one's last nerve, to make one livid, to aggravate, " +
+			"to make one’s blood boil, to exasperate, to get into one's head</i>",
```

Comment:
```suggestion
			"to make one's blood boil, to exasperate, to get into one's head</i>",
```

Please use the regular apostrophe 😄

Reply:
Oops, my mistake for copying and pasting directly from the GH issue 😬 . Thanks @mhkuu for your sharp eyes!